services/online_exam_service/.env

```
@@ -36,7 +36,7 @@ RATE_LIMIT_MAX=100
 # BLOCKCHAIN CONFIG (Giữ nguyên các giá trị của bạn)
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
 GRADE_LEDGER_CONTRACT_ADDRESS=0xFf72998a0e9178Aa2890B82F90bAB2De0Cd118b0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRADE_LEDGER_CONTRACT_ADDRESS key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)